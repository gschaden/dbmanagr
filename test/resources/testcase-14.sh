db-navigator.py -s skiline@localhost/skiline/api_key/669ba310c9fb11e2aa1fd702d6148dbf
