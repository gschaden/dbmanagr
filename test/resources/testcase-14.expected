Id	Title	Subtitle	Autocomplete	
13704947-ceff-3779-9a96-0b647a9a2320	Hikeline	ffea0d34c9cb11e2995da3700476952a	skiline@localhost/skiline/api_key/669ba310c9fb11e2aa1fd702d6148dbf/

